examples/README.md

```diff
@@ -42,7 +42,7 @@ and an API token.
 
 ## Building the image
 
-To build the demo analytic, simply run the commands below from this directory.
+To build the demo analytic, simply run the commands below **from this directory**.
```

Comment:
You emphasized **from this directory** but then changed the code below so that it does NOT operate from the `examples/` directory. Confused.

Others have successfully used this. What was the problem that prompted these changes?

Reply:
Not really, at least not how it makes sense to me. Once you enter into the examples directory you just run the commands one-by-one without erroneously changing directories. I don't see why we would need to go back to the examples directory to run a script that's outside of that directory. That part doesn't make sense to me.

Reply:
<img width="1177" alt="Screen Shot 2019-08-26 at 9 24 52 AM" src="https://user-images.githubusercontent.com/17344791/63693999-73740e00-c7e3-11e9-90a0-3ce1de00db23.png">